packages/core/src/flex-layout/FlexLayout.tsx

```diff
@@ -71,7 +79,7 @@ function parseAlignment(style: string | undefined) {
   return style === "start" || style === "end" ? `flex-${style}` : style;
 }
 
-function parseSpacing(value: number | string | undefined) {
+export function parseSpacing(value: number | string | undefined) {
```

Comment:
We should be careful that we don't export this publicly, I think right now we are